docs/installation/zoom-configuration/zoom-setup-oauth.md

```diff
@@ -4,53 +4,51 @@ description: Steps to configure your Zoom account to work with Mattermost
 
 # Zoom Setup \(OAuth\)
 
-We are going to set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom and used to create meetings and pull user data:
+You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom and use it to create meetings and pull user data.
 
-* Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
+**Note:** If you've been using Zoom prior to v1.4 you likely have a webhook-type app configured in Zoom. Once you switch over to OAuth, the Zoom admin should disable/remove this app prior to creating the new OAuth Zoom app. If the app is not disabled/removed, you may experience issues with meeting update information not being posted.
```

Comment:
Sorry for the misunderstanding. It doesn't matter the type of app (OAuth or JWT), what matters is that you have a legacy webhook app configured, that were working on 1.3 (if I am not mistaken) but on 1.4 are no longer working.

Also, the problem is not so much about removing or not removing the previous one (should be removed so it does not send gibberish to the server), but that you have to configure the webhooks one more time in the right place (explained in the webhooks section).

Reply:
Ah, that makes sense. 

In that case, this information should be moved to: https://mattermost.gitbook.io/plugin-zoom/installation/zoom-configuration as that is the landing page which splits to JWT and OAuth. Thoughts?

Reply:
Sounds perfect :D

Reply:
Done. :)